tests/test_image/test_image_misc.py

```diff
@@ -24,15 +24,29 @@ def test_tensor2imgs():
         tensor = torch.randn(2, 3, 3)
         mmcv.tensor2imgs(tensor)
 
+    # test tensor dim-1
+    with pytest.raises(AssertionError):
+        tensor = torch.randn(2, 4, 3, 3)
+        mmcv.tensor2imgs(tensor)
+
     # test mean length
     with pytest.raises(AssertionError):
         tensor = torch.randn(2, 3, 5, 5)
         mmcv.tensor2imgs(tensor, mean=(1, ))
+        tensor = torch.randn(2, 1, 5, 5)
+        mmcv.tensor2imgs(tensor)
```

Comment:
Would it be better if the dimensionality of the default mean and std automatically matches the input channels?